src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/TestFixtures/Setup/CertificateUtility.cs

```diff
@@ -201,7 +201,7 @@ internal static byte[] DecryptRsaDirectly(byte[] rsaPfx, byte[] ciphertextCek, s
             Debug.Assert(rsaPfx != null && rsaPfx.Length > 0);
             // The rest of the parameters may be invalid for exception handling test cases
 
-#if NET9_0_OR_GREATER
+#if NET9_0
```

Comment:
How did these changes fix the ARM builds? Are the constants defined differently there?

Reply:
It seems the NET9_0_OR_GREATER constant isn't available everywhere as of now. I found that missing in the logs where the build failed.